src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+.body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
```

Comment:
It seems like you are using a non-existent image file in the `background-image` property. The files listed are `staractiv.png` and `starnotactive.png`, but here you reference `star.svg`. Make sure the file path and name are correct.